c2corg_ui/static/js/follow.js

```diff
@@ -57,7 +57,7 @@ app.FollowController = function(appAuthentication, appApi) {
    */
   this.followed = false;
 
-  if (this.auth_.isAuthenticated()) {
+  if (this.canFollow()) {
     this.api_.isFollowing(this.docId).then(function(response) {
       this.followed = response['data']['is_following'];
```

Comment:
concerning the canFollow function - put the // comments up in the commented description section.